app/services/group/seed.rb

```diff
@@ -16,7 +16,7 @@ def validate(tenant)
       true
     end
 
-    def tenant
+    def external_tenant
```

Comment:
Should this method be private? At first I thought it was weird that lines like [line 13](https://github.com/RedHatInsights/catalog-api/pull/706/files#diff-78e962b0126b215546cc382e6b58c21eR13) worked, but then realized that that was a local variable that happened to have the same name as this method used to. Are we using this anywhere else outside of this class?